lib/html-to-vdom.js

```diff
@@ -3,8 +3,23 @@ var parseHTML = require('./parse-html');
 
 module.exports = function initializeHtmlToVdom (VTree, VText) {
     var htmlparserToVdom = createConverter(VTree, VText);
-    return function convertHTML(html) {
+    return function convertHTML(html, options) {
```

Comment:
Sure, I'll make both of those changes and update. Was trying to change as little as possible!